index.js

```diff
@@ -0,0 +1,15 @@
+// This is Default element selector
+const selectElements = selector =>{
+    const element = document.querySelector(selector);
+    if (element) return element;
+    else throw new Error(`something went wrong check if ${selector} is spelt correctly`)
```

Comment:
You meant `not spelt correctly ...`

Reply:
I actually meant if it is `spelt correctly` so of course `if not...` should be crosschecked and see what may be giving the `Error`